docs/readthedocs/models/input/grid/line.md

```diff
@@ -18,7 +18,7 @@ Representation of an AC line.
 | vRated    | kV      | Rated voltage                               |
 
 
-A list with some standard line types can be found here: `Standard Line Types`_
+A list with some standard line types can be found here: [Standard Line Types](#standard-line-types)_
```

Comment:
The underscore character can be removed as well.

(You can find a preview of the changed documentation [here](https://powersystemdatamodel--980.org.readthedocs.build/en/980/models/input/grid/line.html))